package.json

```diff
@@ -17,7 +17,8 @@
     "check:deps": "pnpm --recursive --parallel exec depcheck",
     "check:format": "prettier . --check",
     "check:lint": "turbo run lint --continue -- --quiet",
-    "check:react-compiler": "eslint --no-inline-config --no-eslintrc --ext .cjs,.mjs,.js,.jsx,.ts,.tsx --parser @typescript-eslint/parser --plugin react-compiler --rule 'react-compiler/react-compiler: [warn]' --ignore-path .eslintignore.react-compiler --max-warnings 79 .",
+    "check:react-compiler": "eslint --cache --no-inline-config --no-eslintrc --ext .cjs,.mjs,.js,.jsx,.ts,.tsx --parser @typescript-eslint/parser --plugin react-compiler --rule 'react-compiler/react-compiler: [warn]' --ignore-path .eslintignore.react-compiler --max-warnings 77 .",
+    "report:react-compiler-bailout": "eslint --cache --no-inline-config --no-eslintrc --ext .cjs,.mjs,.js,.jsx,.ts,.tsx --parser @typescript-eslint/parser --plugin react-compiler --rule 'react-compiler/react-compiler: [error,{__unstable_donotuse_reportAllBailouts:true}]' --ignore-path .eslintignore.react-compiler -f ./scripts/reactCompilerBailouts.cjs . || true",
```

Comment:
The reason this command has `__unstable_donotuse_reportAllBailouts:true` is because we want to make sure Million Lint runs on all components that are not optimized by the compiler.

Running the eslint compiler plugin reports 77 issues atm that it considers actionable.
Actionable means its bailing because of a problem with the code, for example reading a `ref.current` during render (which has always been unsafe, and the reason it's not impacting us atm is because we're over-rendering).
There are a lot of cases where the compiler skips a component that's valid. For example it doesn't fully support TypeScript "satisfies" syntax, [here it bails](https://playground.react.dev/#N4Igzg9grgTgxgUxALhAFwJ4AcEAICScEAdgCrZ4C8uA5ADYQCGAJgJbEDmNuAPrWIwBuCGgB1i4hAA8sEGGlzMEAM0ZQ6C5VGJw0rErgBCUNGhIAKLDAhYwASlzBxuXEWJgFrN7mqXrtgDoGFnYOXAB+WmC2Tm5kfiERBwE9MGVWBDACN3IcZ1wYBDRYYlwAHgAjEzNSrxJKYDriAF8APgBhOi8AawBCMoB6KtMSVoBucWbxEGagA), [while here it works.](https://playground.react.dev/#N4Igzg9grgTgxgUxALhAFwJ4AcEAICScEAdgCrZ4C8uA5ADYQCGAJgJbEDmNuAPrWIwBuCGgB1i4hAA8sEGGlzMEAM0ZQ6C5VGJw0rErgBCUNGhIAKLDAhYwASlzBxuXEWJgFrN8gJvyOXGpLa1sAOgYWdg5cAH5aCLZObh8aAWEaO2dcGAQ0WGJcAB4AIxMzAq8SSmBK4gBfAD4AYTovAGsAQkKAelLTEgaAbnE68RA6oA)
These type of issues aren't flagged without the `__unstable_donotuse_reportAllBailouts` option to avoid leading people to refactor code in ways to only please the current version of the compiler, instead of what's best for code readability and for the humans working in the project.

In this case we don't really care or show the bailed out components that aren't actionable, we just gobble it up in a JSON file to setup mutually exclusive source filters between Million Lint and React Compiler.
That way we get intel on which of the components that don't have react errors, but for whatever reason isn't compiled, is worth looking into manually memoizing in the mean time.